src/widgets/splits/Split.cpp

```diff
@@ -635,7 +636,9 @@ void Split::showViewerList()
     for (auto &x : labels)
     {
         auto label = new QListWidgetItem(x);
-        label->setBackgroundColor(this->theme->splits.header.background);
+        label->setBackground(this->theme->splits.header.background);
+        label->setForeground(QColor(169, 112, 255));
```

Comment:
I don't think we use twitch purple anywhere else in the application. It feels out of place to me to just do it in one very specific UI. Someone correct me if I'm wrong or they feel otherwise.

Reply:
@TranRed The color is arbitrary to me, but I think it should be different than the usernames or vice versa. Twitch has white labels and purple usernames. Chatterino does not really have an accent color that I could tell, so I was unsure of what to go with. 

Do we have a theme color that makes sense here?

Reply:
Done

Reply:
I think the closest we have to an accent color is focusedText. But this is just my interpretation. No guarantee it's intended to use in this way.
https://github.com/Chatterino/chatterino2/blob/95c909b33755294a0227ded3c6941ec87d9d50fe/src/singletons/Theme.cpp#L54-L55

If we introduce an accent color it should go into Theme.hpp/cpp though, so it can be reused in other places.

Reply:
In the latest master, there's an `accent` variable you can use from theme - try that out for the color of the "headers"

Reply:
I updated the headers to use the new theme accent color. It looks great on dark theme.

I don't know where this app/team stands on accessibility, but the theme color for text on Light and White themes is not AA or AAA accessible.

Results here:
White https://webaim.org/resources/contrastchecker/?fcolor=00AEEF&bcolor=FFFFFF
Light https://webaim.org/resources/contrastchecker/?fcolor=00AEEF&bcolor=E6E6E6
Dark https://webaim.org/resources/contrastchecker/?fcolor=00AEEF&bcolor=191919
Black https://webaim.org/resources/contrastchecker/?fcolor=00AEEF&bcolor=000000

Reply:
I believe the entire white/light theme family is a bit (i.e. very) orphaned.